loader/src/loader.rs

```diff
@@ -4,7 +4,7 @@ use std::{error::Error, sync::Arc};
 
 /// Loading ID allocated by `atelier-assets` to track loading of a particular asset.
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
-pub struct LoadHandle(pub(crate) u64);
+pub struct LoadHandle(pub u64);
```

Comment:
Why is this public? This allows fabricating fake load handles anywhere.

Reply:
It's for the serde "dummy" implementation, check the amethyst diff linked.

Reply:
I'd give it a pass, but it still smells bad :P Ideally the "mock" tools would simply live in this crate as well.